kinova_gen3_7dof/gen3_7dof.xml

```diff
@@ -0,0 +1,124 @@
+<mujoco model="gen3">
+  <compiler angle="radian" meshdir="assets"/>
+
+  <option integrator="implicitfast"/>
+
+  <default>
+    <!-- kinova defaults -->
+    <default class="kinova">
+      <material specular="0.5" shininess="0.25"/>
+      <joint armature="0.1" damping="1" axis="0 0 1" range="-2.8973 2.8973"/>
+      <general dyntype="none" biastype="affine" ctrlrange="-6.28 6.28" forcerange="-87 87"/>
```

Comment:
should `ctrlrange` on line 11 actually match the `range` on line 10?

Reply:
https://github.com/PickNikRobotics/mujoco_menagerie/pull/1/commits/ae6d82ebd0951554d58229c51831c7b024165f0f